SigningService/src/DataBlockHashing.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SigningService
```

Comment:
This should not exist in the root project directory. Lets move it into \Signers\StrongName\